packages/infra/config/env-config.ts

```diff
@@ -266,6 +266,18 @@ type EnvConfigBase = {
       };
     };
   };
+  surescripts?: {
```

Comment:
why optional? if not optional you can remove the if statement from `secrets-stack.ts` completely